modules/ocf_syslog/files/ocf.conf

```diff
@@ -15,41 +25,47 @@ template(name="ocf-remote" type="list") {
     constant(value="\n")
 }
 
-# template for errored remote log lines
-template(name="ocf-errors" type="list") {
-    constant(value="ERRORED_TAG[")
-    property(name="syslogtag")
-    constant(value="]  MESSAGE[")
-    property(name="timegenerated" dateFormat="rfc3339")
-    constant(value=" ")
-    property(name="hostname")
-    constant(value=":  ")
-    property(name="msg")
-    constant(value="]\n")
+# template for remote docker log filenames
+template(name="ocf-docker-file" type="list") {
+    constant(value="/var/log/remote/services/")
+    property(name="syslogtag" securepath="replace")
+    constant(value="/")
+    property(name="syslogtag" securepath="replace")
+    constant(value=".log")
 }
 
 # template for remote log filenames
 template(name="ocf-remote-file" type="list") {
     constant(value="/var/log/remote/")
-    # in jessie, syslogtag has a weird ":" appended, so we have to regex that out;
-    # the regex bits can be removed when we're totally on stretch.
-    property(name="syslogtag" regex.Expression="^[^:]+" regex.Type="ERE" regex.Match="0" SecurePath="replace")
+    property(name="hostname" securepath="replace")
+    constant(value="/")
+    property(name="programname" caseconversion="lower" securepath="replace")
+    constant(value="/")
+    property(name="programname" caseconversion="lower" securepath="replace")
     constant(value=".log")
 }
 
 # rules for storing the logs
 ruleset(name="ocf-remote") {
-    # This regex was fiendish, since the hyphen must be next to the end bracket
-    # for it to work.
+    # The regex for the syslogtag was fiendish, since the hyphen must be next
+    # to the end bracket for it to work.
     #
     # It uses POSIX ERE (http://www.regular-expressions.info/posixbrackets.html)
     # and has a tester at http://www.rsyslog.com/regex/
     #
     # Make sure this at least matches all the items let through in the regex in
     # the stdin2syslog script so that real logs aren't put in errors.log
```

Comment:
Are we still doing errors.log?

Reply:
No, so this comment is wrong, I'll fix it. I was tempted to keep `errors.log`, but the main problem is that I don't think anything would get put into it unless it doesn't have a hostname set in the request, which I don't see happening.